OsmAnd-java/src/main/java/net/osmand/binary/RouteDataObject.java

```diff
@@ -765,23 +765,23 @@ public boolean isExitPoint() {
 //		return false;
 //	}
 
-	public String getExitName() {
-		if (pointNames != null && pointNameTypes != null) {
-			int pnSz = pointNames.length;
-			for (int i = 0; i < pnSz; i++) {
-				String[] point = pointNames[i];
-				if (point != null) {
-					int pSz = point.length;
-					for (int j = 0; j < pSz; j++) {
-						if (pointNameTypes[i][j] == region.nameTypeRule) {
-							return point[j];
-						}
-					}
-				}
-			}
-		}
-		return null;
-	}
+//	public String getExitName() {
```

Comment:
no need to delete exit name might be useful in future

Reply:
Ok